src/js/component.js

```diff
@@ -360,21 +362,19 @@ Component.prototype.getChild = function(name){
  *       }
  *     });
  *
- * @param {String|vjs.Component} child The class name or instance of a child to add
+ * @param {String|Component} child The class name or instance of a child to add
  * @param {Object=} options Options, including options to be passed to children of the child.
- * @return {vjs.Component} The child component (created by this process if a string was used)
+ * @return {Component} The child component (created by this process if a string was used)
  * @suppress {accessControls|checkRegExp|checkTypes|checkVars|const|constantProperty|deprecated|duplicate|es5Strict|fileoverviewTags|globalThis|invalidCasts|missingProperties|nonStandardJsDocs|strictModuleDepCheck|undefinedNames|undefinedVars|unknownDefines|uselessCode|visibility}
  */
-Component.prototype.addChild = function(child, options){
+Component.prototype.addChild = function(child, options={}){
   let component, componentName;
   // If child is a string, create nt with options
   if (typeof child === 'string') {
     let componentName = child;
 
-    // Make sure options is at least an empty object to protect against errors
-    if (!options || options === true) {
-      options = {};
-    }
+    // Apparently sometimes options is just true, so we need that to make sure we still convert options to an object.
```

Comment:
"Apparently" :-/

Yeah, you can do:
```
{
  children: {
    controlBar: true,
    posterImage: false
  }
}
```

The `false` bit is useful. I think the true might have just been for symmetry. Now that you should be defining children with arrays I think we can ditch the `true` option (but not `false`).